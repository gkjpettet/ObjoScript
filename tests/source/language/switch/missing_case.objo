var consider = 5

// Expect error: At least one case is required.
switch consider {
else {}
}