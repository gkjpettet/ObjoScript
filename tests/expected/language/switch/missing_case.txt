A switch statement must include at least one case.